react/features/app/components/AbstractApp.js

```diff
@@ -134,11 +137,13 @@ export class AbstractApp extends Component {
 
         if (route) {
             return (
-                <Provider store = { this._getStore() }>
-                    {
-                        this._createElement(route.component)
-                    }
-                </Provider>
+                <I18nextProvider i18n = { i18n }>
```

Comment:
I think this can be simplified by passing the i18n prop to the `Provider` below, since `I18nextProvider` looks like a simple wrapper.